### PR TITLE
Added description.rst to generate repository README

### PR DESCRIPTION
**Pull request by cesarlr**
See https://github.com/coopiteasy/vertical-cooperative/pull/91
I am not sure if I can do
`oca-gen-addon-readme --repo-name vertical-cooperative --branch 12.0 --org-name coopiteasy --addon-dir easy_my_coop`
before merging this pull requesting

### CODE REVIEW

**Comment by sseerrggii** [comment]
@robinkeunen  :wave:

File: easy_my_coop/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Vertical Cooperative
```
Comment (robinkeunen) [inline easy_my_coop/DESCRIPTION.rst]: The section title will be generated by the oca-gen command.

**Review by robinkeunen — COMMENTED** [review]
Thanks @cesarlr and sorry for the delay, it went under my radar. I'll see if I can better configure my notifications.

Please remove the section title and do run `oca-gen-addon-readme` command and I'll merge this.

**Comment by enricostano** [comment]
@robinkeunen better now?

**Comment by robinkeunen** [comment]
Yes, thanks!

**Pull request merged**